common/js/src/logging/debug-dump.js

```diff
@@ -183,7 +183,8 @@ function dump(value) {
   // Note that goog.dom.is* methods are not used because many of them may
   // produce thorny false positives.
   //
-  // TODO(emaxx): Think about a proper solution that deals with cyclic references.
+  // TODO(emaxx): Think about a proper solution that deals with cyclic
```

Comment:
Sorry, committed these unrelated changes by mistake. But there's not much of them anyway.